InvenTree/order/templates/order/purchase_order_detail.html

```diff
@@ -179,6 +179,9 @@ <h4>{% trans "Order Notes" %}</h4>
         currency: '{{ order.supplier.currency }}',
         {% endif %}
         {% endif %}
+        {% if order.target_date %}
+        target_date: '{{ order.target_date|date:'Y-m-d' }}',
```

Comment:
Use the "render_date" template tag here - it will automatically render dates according to the user's preference.

Reply:
Good to know, I've updated the code